extensions/shared/components/audio-player/index.jsx

```diff
@@ -72,8 +74,15 @@ function AudioPlayer( {
 
 	useEffect( () => {
 		const audio = audioRef.current;
-		// Initialize MediaElement.js.
```

Comment:
Does using the store here resolve the issue with `MediaElement.js` described [here](https://github.com/Automattic/jetpack/blob/master/extensions/shared/components/audio-player/README.md#potential-issues)? 

> Currently the component is a wrapper around a MediaElement.js component which does its own event handling. It's possible therefore, that the state of the player can get out of sync with the state of the controlling component. We have seen an example of this caused by rapid clicking on the track progress bar, which was solved by debouncing the events.

Reply:
Not sure whether it could be related. Honestly, I haven't seen these _out of sync_ issues while testing and developing.

Reply:
Thanks! Should the `// Initialize MediaElement.js.` comment stay for now?

Reply:
yes, as long as we create a new MediaElement instance we should comment that. We don't have planned to remove the MediaElement in the short time AFAIK.